src/client/vuecreatejob.js

```diff
@@ -0,0 +1,206 @@
+
+// loads Jobs
+// const server = "https://masteringjs-job-board.azurewebsites.net";
+const server = "http://localhost:7071";
+const payment = server+'/api/stripeCheckout';
+
+const app = new Vue({
+  data: () => ({
+    company: null,
+    logo: null,
+    title: null,
+    location: 'Anywhere',
+    sticky: false,
+    description: null,
+    url: null,
+    instructions: null,
+    email: null,
+    feedback: null,
+    invoiceAddress: null,
+    invoiceNotes: null,
+    tags: 'like,this',
+    displayImage: true,
+    previewImage: null,
+    loading: false,
+    isActive: false
+  }),
+  updated() {
+    console.log(new Date(), 'State Change:', this.$data);
+  },
+  mounted() {
+    window.$saveState = () => {
+      window.localStorage.setItem('__state', JSON.stringify(this.$data));
+    };
+    window.$loadState = () => {
+      const saved = JSON.parse(window.localStorage.getItem('__state'));
+      if (!saved) {
+        throw new Error('No saved data!');
+      }
+      Object.assign(this.$data, saved);
+    };
+    window.$clearState = () => {
+      window.localStorage.setItem('__state', '');
+    };
+  },
+  methods: {
+    async postJob() {
+      this.loading = true;
+      this.tags = this.tags.split(",");
+     let clientid = await axios.post(server + '/api/createJob', {
```

Comment:
There's no need to have separate `createJob` and `stripeCheckout` functions. Might as well just have `createJob` do the work of creating the stripe checkout session